<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1623301882163' stop='1623301917855'>
    <name>Super Calculator  </name>
    <title>Super Calculator  </title>
    <test-cases>
        <test-case start='1623301882164' status='passed' stop='1623301893822'>
            <name>Validate Addition of two numbers</name>
            <title>Validate Addition of two numbers</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1623301893822' status='passed' stop='1623301900163'>
            <name>Validate Substraction of two numbers</name>
            <title>Validate Substraction of two numbers</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1623301900163' status='passed' stop='1623301906484'>
            <name>Validate Multiplication of two numbers</name>
            <title>Validate Multiplication of two numbers</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1623301906484' status='passed' stop='1623301912656'>
            <name>Validate Division of two numbers</name>
            <title>Validate Division of two numbers</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1623301912656' status='failed' stop='1623301917854'>
            <name>Validate Addition of two numbers</name>
            <title>Validate Addition of two numbers</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Expected '5' to be 5.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (D:\synechron_07_06_protractor\javascriptTraining\PROTRACTOR\development.js:59:57)
                    at C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
                    at new ManagedPromise (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
                    at ControlFlow.promise (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
                    at schedulerExecute (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
                    at TaskQueue.execute_ (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2974:25
                    at C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>